compiler/rustc_metadata/src/native_libs.rs

```diff
@@ -472,7 +472,9 @@ impl<'tcx> Collector<'tcx> {
                 Abi::Fastcall { .. } => {
                     DllCallingConvention::Fastcall(self.i686_arg_list_size(item))
                 }
-                // Vectorcall is intentionally not supported at this time.
+                Abi::Vectorcall { .. } => {
```

Comment:
Would x64 vectorcall support be a follow-up change?

Reply:
Potentially: `vectorcall` is still unstable in Rust and is only targeted at i686, so if someone were to decide to expand that to x64 then they would also need to make sure to add support in `raw-dylib` as well.